.bacon.yml

```diff
@@ -1,99 +1,110 @@
 test_suites:
```

Comment:
Moving these up on `0.0` until we merge back into `master`

Reply:
Would the the tasks be ordered as indexed in this YAML if `sort_order` is not specified?

Reply:
I also figured out the travis approach may not work, since 0.0 is a protected branch. So maybe we can set unnecessary tasks as optional before changes hit master.

Reply:
Yeah. Did that (see below) conservatively. Lmk which tasks you think are OPTIONAL on 0.0